adding "pi-pins" to this template

Hi
I am trying to use the GPIOs so I added "pi-pins". my dependencies in my package.json look like this 
```
"dependencies": {
        "bootstrap": "^4.0.0-alpha.4",
        "electron": "^1.4.13",
        "firebase": "^3.6.8",
        "jquery": "^3.1.0",
        "moment": "^2.17.1",
        "pi-pins": "~1.0.2",
        "require-rebuild": "^1.2.8",
        "tether": "^1.3.7",
        "weather-js": "^1.0.2"
    }
```

when I tried to run it on my RPI3 i got this error message:

> [194:0201/234145:INFO:CONSOLE(83)] "Uncaught Error: Module version mismatch. Expected 50, got 48.", source: /usr/src/app/node_modules/bindings/bindings.js (83)

if I Remove the code below (which is in my index.js) the error goes away:

```
if (isPi) {
        var GPIO = require('pi-pins');
    }
```

I am doing something wrong?

@derrickaxel  How did you revert to electron 1.2.5? When I tried `"electron": "1.2.5",` I got the following error:
```
npm ERR! Linux 4.7.3-coreos-r3
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "--unsafe-perm" "--production"
npm ERR! node v6.9.2
npm ERR! npm  v3.10.9
npm ERR! code ETARGET

npm ERR! notarget No compatible version found: electron@1.2.5
npm ERR! notarget Valid install targets:
npm ERR! notarget 1.5.0, 1.4.15, 1.4.14, 1.4.13, 1.4.12, 1.4.11, 1.4.10, 1.4.8, 1.4.7, 1.4.6, 1.4.5, 1.4.4, 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.13, 1.3.12, 1.3.10, 1.3.9, 1.3.8, 1.3.7, 1.3.6, 1.3.5, 1.3.4, 1.3.3, 1.3.2, 1.3.1, 0.4.1, 0.4.0, 0.3.0, 0.2.1, 0.2.0, 0.1.2, 0.1.1, 0.1.0
npm ERR! notarget
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'clockos'
npm ERR! notarget

npm ERR! Please include the following file with any support request:
npm ERR!     /usr/src/app/npm-debug.log
Removing intermediate container b22cb931e8cc
Build failed: BuildError: The command '/bin/sh -c JOBS=MAX npm install --unsafe-perm --production       && npm cache clean' returned a non-zero code: 1
```
It isn't finding electron 1.2.5

That's weird, I believe I'm using node 6.9.2 because of this first line in the dockerfile `FROM resin/%%RESIN_MACHINE_NAME%%-node:6.9.2`, but when I try it using electron 1.2.5 with `^` my build chooses 1.4.15 and when I use it without it it says:
```
npm ERR! notarget No compatible version found: electron@1.2.5
npm ERR! notarget Valid install targets:
npm ERR! notarget 1.5.0, 1.4.15, 1.4.14, 1.4.13, 1.4.12, 1.4.11, 1.4.10, 1.4.8, 1.4.7, 1.4.6, 1.4.5, 1.4.4, 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.13, 1.3.12, 1.3.10, 1.3.9, 1.3.8, 1.3.7, 1.3.6, 1.3.5, 1.3.4, 1.3.3, 1.3.2, 1.3.1, 0.4.1, 0.4.0, 0.3.0, 0.2.1, 0.2.0, 0.1.2, 0.1.1, 0.1.0
```

--- a/Dockerfile.template
+++ b/Dockerfile.template
@@ -45,7 +45,7 @@ COPY ./app/package.json ./
 
 # Install npm modules for the application
 RUN JOBS=MAX npm install --unsafe-perm --production \
-	&& npm cache clean &&
+	&& npm cache clean && node_modules/.bin/electron-rebuild
 
 # Move app to filesystem
 COPY ./app ./

--- a/Dockerfile.template
+++ b/Dockerfile.template
@@ -1,4 +1,4 @@
-FROM resin/%%RESIN_MACHINE_NAME%%-node:6.9.2
+FROM resin/%%RESIN_MACHINE_NAME%%-node:6.9
 
 # debian httpredir mirror proxy often ends up with 404s - editing source file to avoid it
 RUN sed -i "s!httpredir.debian.org!`curl -s -D - http://httpredir.debian.org/demo/debian/ | awk '/^Link:/ { print $2 }' | sed -e 's@<http://\(.*\)/debian/>;@\1@g'`!" /etc/apt/sources.list
@@ -37,21 +37,18 @@ RUN echo "#!/bin/bash" > /etc/X11/xinit/xserverrc \
   && echo "" >> /etc/X11/xinit/xserverrc \
   && echo 'exec /usr/bin/X -s 0 dpms -nocursor -nolisten tcp "$@"' >> /etc/X11/xinit/xserverrc
 
-# Save source folder
-RUN printf "%s\n" "${PWD##}" > SOURCEFOLDER
-
 # Move to app dir
 WORKDIR /usr/src/app
 
 # Move package.json to filesystem
-COPY "$SOURCEFOLDER/app/package.json" ./
+COPY ./app/package.json ./
 
 # Install npm modules for the application
 RUN JOBS=MAX npm install --unsafe-perm --production \
-	&& npm cache clean
+	&& npm cache clean &&
 
 # Move app to filesystem
-COPY "$SOURCEFOLDER/app" ./
+COPY ./app ./
 
 ## uncomment if you want systemd
 ENV INITSYSTEM on

--- a/app/main.js
+++ b/app/main.js
@@ -1,4 +1,3 @@
-require('require-rebuild')();
 const electron = require('electron');
 const path = require('path');
 

--- a/app/package.json
+++ b/app/package.json
@@ -19,8 +19,8 @@
     "js"
   ],
   "dependencies": {
-    "electron": "^1.4.13",
-    "require-rebuild": "^1.2.8"
+    "electron": "^1.4.15",
+    "electron-rebuild": "^1.5.7"
   },
   "author": "Carlo Maria Curinga",
   "license": "Apache-2.0",

